tests/compile-fail/exact_div3.rs

```diff
@@ -1,5 +1,5 @@
 #![feature(core_intrinsics)]
 fn main() {
     // signed divison with a remainder
-    unsafe { std::intrinsics::exact_div(-19i8, 2); } //~ ERROR Scalar(0xed) cannot be divided by Scalar(0x02) without remainder
+    unsafe { std::intrinsics::exact_div(-19i8, 2); } //~ ERROR 237 cannot be divided by 2 without remainder
```

Comment:
@oli-obk I am not sure if this is the desired printing here... we are printing a negative number as a positive one.